src/platform/electron/protocol-handler.ts

```diff
@@ -0,0 +1,11 @@
+export function subscribeToDeepLinkURLs(callback: (url: string) => void) {
+  if (!window.electron) {
+    throw new Error("No electron runtime context available.")
+  }
+
+  return window.electron.subscribeToIPCMain("deeplink:url", (event, args) => {
```

Comment:
Let’s rename `args` to `url`. Btw, regarding the line below: Is there a real risk that the IPC message body (`args`) is not set?